lib/create-options.js

```diff
@@ -10,6 +10,7 @@ function createOptions(cwd, context) {
   options.env = Object.create(process.env);
   options.env['npm_loglevel'] = 'error';
   options.env['npm_config_tmp'] = context.npmConfigTmp;
+  options.shell = true;
```

Comment:
do we need to this for just windows? Is there a side effect to have this always set?